NitroxModel/Platforms/Store/Steam.cs

```diff
@@ -85,17 +85,17 @@ public async Task<ProcessEx> StartGameAsync(string pathToGameExe, int steamAppId
 #if DEBUG // Needed to start multiple SN instances, but Steam Overlay doesn't work this way so only active for devs
             return ProcessEx.Start(
                 pathToGameExe,
-                new[] { ("SteamGameId", steamAppId.ToString()), ("SteamAppID", steamAppId.ToString()), (NitroxUser.LAUNCHER_PATH_ENV_KEY, NitroxUser.LauncherPath) },
+                [("SteamGameId", steamAppId.ToString()), ("SteamAppID", steamAppId.ToString()), (NitroxUser.LAUNCHER_PATH_ENV_KEY, NitroxUser.LauncherPath)],
                 Path.GetDirectoryName(pathToGameExe),
                 launchArguments
             );
-#endif
-
+#else
             return new ProcessEx(Process.Start(new ProcessStartInfo
             {
                 FileName = GetExeFile(),
                 Arguments = $"""-applaunch {steamAppId} -nitrox "{NitroxUser.LauncherPath}" {launchArguments}"""
             }));
+#endif
```

Comment:
This code is already changed on the linux branch, no point doing anything on master with it.